detox/runners/jest/JestCircusEnvironment.js

```diff
@@ -1,4 +1,6 @@
-const NodeEnvironment = require('jest-environment-node');
+const maybeNodeEnvironment = require('jest-environment-node'); // eslint-disable-line node/no-unpublished-require
+// @ts-ignore
+const NodeEnvironment = maybeNodeEnvironment.default || maybeNodeEnvironment;
```

Comment:
Though a very subtle one - still a dup. Would it make sense to import through a custom file (e.g. `detox/somewhere/NodeEnvironment.js`, instead?

Reply:
No sense. This integration is dropped in Detox 20

Reply:
Because of this, you mean? -
>Generally we recommend not to stay on outdated Jest versions for too long, e.g. `jest@27.2.5` will be the minimal version supported by Detox 20.

Reply:
No, this integration is very old and will be dropped regardless. The next minimal version requirement itself is motivated by the streamline reporter monkey-patch.

The maybe/default trick will remain because we'll be supporting both Jest 27/28